js/terms.json

```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Bundled contribution",
+    "definition": "A contribution forwarded to a reporting committee by a lobbyist/registrant or lobbyist/registrant PAC, or received by a reporting committee and credited to a lobbyist/registrant or lobbyist/registrant PAC. See 11 CFR 104.22(a)(6)."
+  },
+  {
+    "term": "Campaign traveler",
+    "definition": "Any candidate traveling in connection with an election for federal office, or any individual traveling in connection with an election for federal office on behalf of a candidate or political committee; or any member of the news media traveling with a candidate. 11 CFR 100.93(a)(3)(i)."
+  },
+  {
+    "term": "Clearly identified candidate",
+    "definition": "A candidate is clearly identified when his or her name, nickname, photograph or drawing appears, or when his or her identity is otherwise apparent through an unambiguous reference such as “the President,” “your Congressman,” or “the incumbent,” or through an unambiguous reference to his or her status as a candidate such as “the Democratic presidential nominee” or “the Republican candidate for Senate in the State of Georgia.” 11 CFR 100.17."
+  },
+  {
+    "term": "Commercial vendor",
+    "definition": "Any person providing goods or services to a candidate or political committee whose usual and normal business involves the sale, rental, lease or provision of those goods or services. 11 CFR 116.1(c)."
+  },
+  {
+    "term": "Conduit or intermediary",
+    "definition": "Any person who receives and forwards an earmarked contribution to a candidate or a candidate’s authorized committee. 11 CFR 110.6(b)(2)."
+  },
+  {
+    "term": "Connected organization",
+    "definition": "An organization that uses its treasury funds to establish, administer or solicit contributions to a separate segregated fund.  11 CFR 100.6(a)."
+  },
+  {
+    "term": "Coordinated",
+    "definition": "Made in cooperation, consultation or concert with, or at the request or suggestion of, a candidate, a candidate’s authorized committee or their agents, or a political party committee or its agents. 11 CFR 109.20(a)."
+  },
+  {
+    "term": "Coordinated communication",
+    "definition": "A communication that satisfies a three-pronged test: <ul class='list--numbered'><li>The communication must be paid for by a person other than a federal candidate, authorized committee, or a political party committee, or any agents of the aforementioned entities with whom the communication is coordinated.</li><li>One or more of the five content standards set forth in 11 CFR 109.21(c) must be satisfied; and</li><li>One or more of the five conduct standards set forth in 11 CFR 109.21(d) must be satisfied.</li></ul>A payment for a communication satisfying all three prongs is an in-kind contribution to the candidate or political party committee with which it was coordinated. 11 CFR 109.21."
```

Comment:
Do list items need to be separated onto their own line? Or does that markup happen in-line like this?